src/gradata/hooks/inject_brain_rules.py

```
@@ -0,0 +1,61 @@
+    "timeout": 10000,
+}
+
+MAX_RULES = 10
+MIN_CONFIDENCE = 0.60
+
+
+def _score(lesson) -> float:
+    """Score a lesson dict or Lesson object for injection priority."""
+    conf = lesson["confidence"] if isinstance(lesson, dict) else lesson.confidence
+    state = lesson["state"] if isinstance(lesson, dict) else lesson.state.name
+    state_str = state if isinstance(state, str) else state
```

Ruff (0.15.9)

[warning] Useless `if`-`else` condition

(RUF034)